poll/poll.py

```diff
@@ -159,9 +159,9 @@ def gather_items(data, result, noun, field, image=True):
             else:
                 items.append([key, label])
 
-        if not len(items) > 1:
+        if not items:
```

Comment:
@Kelketek nit: not related to current changes, but this method looks too big. I believe it is a result of breaking single responsibility principle - not only it gathers items from data dict, but also validates them. It might make sense to split it into smaller chunks with single responsibility